corehq/apps/app_manager/models.py

```diff
@@ -2117,7 +2125,7 @@ class Module(ModuleBase, ModuleDetailsMixin):
     referral_list = SchemaProperty(CaseList)
     task_list = SchemaProperty(CaseList)
     parent_select = SchemaProperty(ParentSelect)
-
+    search_config = SchemaListProperty(CaseSearch)
```

Comment:
Instead of being a list of properties this should probably be a nested object that contains the list so that you can also have a field for things like title